fixture-monkey-kotlin/src/main/kotlin/com/navercorp/fixturemonkey/kotlin/introspector/PrimaryConstructorArbitraryIntrospector.kt

```diff
@@ -67,7 +70,16 @@ class PrimaryConstructorArbitraryIntrospector : ArbitraryIntrospector {
                     val generatedByParameters = mutableMapOf<KParameter, Any?>()
                     for (parameter in constructor.parameters) {
                         val resolvedArbitrary = resolveArbitrary(parameter, arbitrariesByPropertyName)
-                        generatedByParameters[parameter] = resolvedArbitrary
+
+                        generatedByParameters[parameter] =
+                            if (parameter.type.jvmErasure != Duration::class) {
+                                resolvedArbitrary
+                            } else {
+                                if (resolvedArbitrary is Long) resolvedArbitrary.toDuration(
+                                    DurationUnit.values().random()
+                                )
+                                else Duration.ZERO
+                            }
```

Comment:
Can you explain why these lines are needed?
What if there are problems without these lines?

Reply:
There is no possibility of exposure up to that value. 

We added it to reduce negative user experience for unexpected cases. Now that I think about it, it sounds like a situation where passing Duration in a `CombinableArbitrary` could cause problems. 

I think that's more of a problem, so I'll remove it. ☺️

Reply:
@this-is-spear 
It fails because the generated `long` is not valid to convert as a `Duration` type.

<img width="1004" alt="image" src="https://github.com/naver/fixture-monkey/assets/10272119/de1075d2-e8e2-45b4-953b-1047d82a7b05">

The sample value from `Arbitraries.longs().between(Long.MIN_VALUE, 0)` would intermittently fail because of the `Duration` precondition.

I think it is better to change the logic below to return null instead of `arbitrariesByPropertyName[parameter.name]` if `value class` throws an exception

```kotlin
 if (parameterKotlinType.isValue) {
                parameterKotlinType.primaryConstructor!!.isAccessible = true
                parameterKotlinType.primaryConstructor!!.call(arbitrariesByPropertyName[parameter.name])
 }
```